UCSRest/src/test/resources/application.properties

```diff
@@ -4,31 +4,31 @@ ucf.general.base-uri=http://localhost:9998
 ucf.general.journal-dir=/tmp/ucf

```

Comment:
Is there a way that we can inject these classes per annotation, instead of listing them in this properties file?
Or do a package scan?

Reply:
Well, yes it's possible and desirable.
If we want to have a sort of plugin system of each component we could scan the various packages and classes to find which of these implement a particular interface or use an annotation.
Then we will map a keyword to the class that has been found and match it with the selected keyword in the properties file.

It's a very good observation for a future task!

Reply:
OK can you please create a task in Jira for it ?
Also I am done with review. May be Antonio should also take a look since there are so many files I 
might have missed one or two!!

Reply:
ok!